Sources/Inject/Integrations/SwiftUI.swift

```diff
@@ -19,14 +15,29 @@ public extension SwiftUI.View {
     }
 }
 
+@propertyWrapper
+public struct ObserveInjection: DynamicProperty {
+    @ObservedObject private var iO = Inject.observer
+    public init() {
+        _ = Inject.load
```

Comment:
Is there a reason why you moved the loading of the injection bundle here? People's code that doesn't use the property wrapper just yet will stop working.